### DISCUSSION TITLE
Can't access full Retroarch menu in RGBPI Os4

### DISCUSSION THREAD

**Discussion by wonkastatic** [forkymcforkface/RGBPi-Extra#23] [Q&A]
Tried in every way.
I need to modify retroarch.cfg, but even via ftp it doesn't work.
It looks like it always overwrite to original file where full access in set as false.
Even installing RGBI extra doesn't help.
Please support me.
Thanks in advance

**Comment by forkymcforkface** [comment]
OS4 is built to override the default retroarch.cfg, I made a tweak where you can modify the global config within sys_override folder.
https://github.com/forkymcforkface/RGBPi-Extra/wiki/Retroarch-Options
